### PR TITLE
PXWEB2-59 Added spacing

### PR DESCRIPTION
**Pull request by oyessb**
Added tokens for spacing. 
Added build.js for building style dictionay with different configuration. 
Changed github action to run build.js.

build.js is created with help from ChatGPT 4.

### CODE REVIEW

**Review by PerIngeVaaje — APPROVED** [review]
OK

File: libs/pxweb2-ui/style-dictionary/build.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    source: [
+      './libs/pxweb2-ui/style-dictionary/src/global-tokens/spacing.json',
+    ],
+    platforms: {
+      scss: {
+        transformGroup: 'scss',
+        buildPath: './libs/pxweb2-ui/style-dictionary/dist/scss/',
+        files: [
+          {
+            destination: 'global.scss',
```
Comment (MikaelNordberg) [inline libs/pxweb2-ui/style-dictionary/build.js:48]: Should we give global.scss another name considering the discussion of yesterday?

**Review by MikaelNordberg — APPROVED** [review]
Maybe change name of global.scss. Otherwise it looks good.

File: package.json [hunk 2]
```diff
@@ -3,11 +3,11 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "start": "npx nx run pxweb2:serve",
+    "start": "npm run build-style-dictionary && nx run pxweb2:serve",
```
Comment (runejo) [inline package.json:6]: There is also a `prestart` script if you want to make it more tidy https://docs.npmjs.com/cli/v10/using-npm/scripts

**Pull request merged**